recipes/cminpack/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "cminpack" %}
+{% set version = "1.3.6" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/devernay/{{ name }}/archive/v{{ version }}.zip
+  sha256: 99865735c21b8c035acfcf9390e7e69efacc6389863044e516b0ae57c8c539f5
+  patches:
+    # The following patch is already in `master` upstream, so can probably
+    # be removed for next version
+    - 0001-Fix-compilation-of-examples-tlmdif_-on-Windows.patch  # [win]
+
+build:
+  skip: true  # [win and vc<14]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}  # Needed because CMake project is not declared as C
+    - cmake
+    - ninja
+
+test:
+  commands:
+    - if not exist %LIBRARY_LIB%\\cminpack.lib exit 1  # [win]
+    - if not exist %LIBRARY_BIN%\\cminpack.dll exit 1  # [win]
+    - if not exist %LIBRARY_INC%\\cminpack-1\\cminpack.h exit 1  # [win]
+    - if not exist %LIBRARY_INC%\\cminpack-1\\minpack.h exit 1  # [win]
+    - if not exist %LIBRARY_PREFIX%\\share\\cmake\\CMinpack\\CMinpackConfig.cmake exit 1  # [win]
+
+    - test -a $PREFIX/lib/libcminpack$SHLIB_EXT  # [unix]
+    - test -f $PREFIX/include/cminpack-1/cminpack.h  # [unix]
+    - test -f $PREFIX/include/cminpack-1/minpack.h  # [unix]
+    - test -f $PREFIX/share/cmake/CMinpack/CMinpackConfig.cmake  # [unix]
+
+about:
+  home: http://devernay.free.fr/hacks/cminpack/index.html
+  license: MINPACK  # Custom BSD-like
+  license_family: BSD
+  summary: 'CMinpack includes software for solving nonlinear equations and nonlinear least squares problems'
+  license_file: CopyrightMINPACK.txt
+  dev_url: https://github.com/devernay/cminpack
+
+extra:
+  recipe-maintainers:
+    - tadeu
+    - xantares
```

Comment:
@xantares, do you want to be a maintainer?

Reply:
@xantares, since you didn't answer and I need an explicit approval for maintainers, I removed you from the list, but feel free to join any time :)